### PR TITLE
rename cyclejs/cycle-core references to cyclejs/core

### PR DESCRIPTION
**Pull request by alkhe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,4 +8,4 @@ Built with Jekyll
 
 ## Why are Issues unavailable?
 
-We use only one repository for issues. [**Open the issue at Cycle Core repo.**](https://github.com/cyclejs/cycle-core/issues)
+We use only one repository for issues. [**Open the issue at Cycle Core repo.**](https://github.com/cyclejs/cycle/issues)
```
Comment (staltz) [inline README.md]: Isn't it `cyclejs/CORE`?

File: _layouts/blank.html [hunk 2]
```diff
@@ -42,7 +42,7 @@
     </script>
     <script>
       ((window.gitter = {}).chat = {}).options = {
-        room: 'cyclejs/cycle-core'
+        room: 'cyclejs/cycle'
```
Comment (staltz) [inline _layouts/blank.html]: `cyclejs/CORE`?

File: _posts/2015-01-21-community.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ To engage with the community around Cycle.js, follow these resources:
 
 * The GitHub organization, [cyclejs](https://github.com/cyclejs), is the default location for finding most of the packages you'll need.
 * Ask, "_How do I...?_" questions in Gitter: [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/staltz/cycle)
-* Report bugs, discuss, and propose significant changes as a [GitHub issue](https://github.com/cyclejs/cycle-core/issues).
+* Report bugs, discuss, and propose significant changes as a [GitHub issue](https://github.com/cyclejs/cycle/issues).
```
Comment (staltz) [inline _posts/2015-01-21-community.md]: `cyclejs/CORE`?

File: _posts/2015-01-21-community.md [hunk 4]
```diff
@@ -17,7 +17,7 @@ Visit [Awesome Cycle.js](https://github.com/vic/awesome-cyclejs) to see a curate
 
 - This project is a grateful recipient of the [Futurice Open Source sponsorship program](http://futurice.com/blog/sponsoring-free-time-open-source-activities).
 - [@TylorS](https://github.com/TylorS) and [@Frikki](https://github.com/Frikki/) for fantastic contributions and involvement with Cycle.js.
-- [@dobrite](https://github.com/dobrite) for [boilerplate reduction ideas](https://github.com/cyclejs/cycle-core/issues/56).
+- [@dobrite](https://github.com/dobrite) for [boilerplate reduction ideas](https://github.com/cyclejs/cycle/issues/56).
```
Comment (staltz) [inline _posts/2015-01-21-community.md]: `cyclejs/CORE`?

File: _posts/2015-01-22-documentation.md [hunk 5]
```diff
@@ -3,7 +3,7 @@ title:  "Documentation"
 tags: chapters
 ---
 
-## Cycle *Core* [v6.0.0](https://github.com/cyclejs/cycle-core/releases/tag/v6.0.0) API: `Cycle` object
+## Cycle *Core* [v6.0.0](https://github.com/cyclejs/cycle/releases/tag/v6.0.0) API: `Cycle` object
```
Comment (staltz) [inline _posts/2015-01-22-documentation.md]: `cyclejs/CORE`?

File: _posts/2015-01-31-getting-started.md [hunk 6]
```diff
@@ -141,7 +141,7 @@ This example portrays the most common problem-solving pattern in Cycle.js: formu
 
 <h2 id="cyclejs-as-a-script">Cycle.js as a script</h2>
 
-In the rare occasion you need Cycle.js as a standalone JavaScript file, you can download them on the [Releases](https://github.com/cyclejs/cycle-core/releases) page at GitHub.
+In the rare occasion you need Cycle.js as a standalone JavaScript file, you can download them on the [Releases](https://github.com/cyclejs/cycle/releases) page at GitHub.
```
Comment (staltz) [inline _posts/2015-01-31-getting-started.md]: `cyclejs/CORE`?

File: _posts/2015-01-31-getting-started.md [hunk 7]
```diff
@@ -141,7 +141,7 @@ This example portrays the most common problem-solving pattern in Cycle.js: formu
...
 
-- Download the latest [Cycle Core](https://github.com/cyclejs/cycle-core/releases)
+- Download the latest [Cycle Core](https://github.com/cyclejs/cycle/releases)
```
Comment (staltz) [inline _posts/2015-01-31-getting-started.md]: `cyclejs/CORE`?

File: index.html [hunk 8]
```diff
@@ -10,7 +10,7 @@
         <h2>A functional and reactive JavaScript framework for cleaner code</h2>
         <h3><a href="/getting-started.html">Read the docs</a></h3>
         <h4><a href="http://widdersh.in/tricycle/">Or try it live!</a></h4>
-        <iframe class="hero-gh-star" src="https://ghbtns.com/github-btn.html?user=cyclejs&repo=cycle-core&type=star&count=true" frameborder="0" scrolling="0" width="170px" height="20px"></iframe>
+        <iframe class="hero-gh-star" src="https://ghbtns.com/github-btn.html?user=cyclejs&repo=core&type=star&count=true" frameborder="0" scrolling="0" width="170px" height="20px"></iframe>
```
Comment (staltz) [inline index.html:13]: This one is right :D

**Comment by alkhe** [comment]
Wow, my bad. Not sure what I was doing there.

**Comment by alkhe** [comment]
Sorry for the inconvenience. Fixed those, and did the same for http, history, and examples.

**Comment by staltz** [comment]
Thanks for this PR :)

**Pull request merged**